src/xpk/commands/storage.py

```diff
@@ -0,0 +1,112 @@
+"""
+Copyright 2024 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from argparse import Namespace
+
+from kubernetes import client as k8s_client
+from kubernetes.client.rest import ApiException
+
+from ..core.core import (
+    setup_k8s_env,
+    update_cluster_with_gcsfuse_driver_if_necessary,
+    update_cluster_with_workload_identity_if_necessary,
+)
+from ..core.storage import (
+    STORAGE_CRD_KIND,
+    XPK_API_GROUP_NAME,
+    XPK_API_GROUP_VERSION,
+    create_storage_instance,
+    get_storage,
+    install_storage_crd,
+    list_storages,
+    print_storages_for_cluster,
+)
+from ..utils import apply_kubectl_manifest, xpk_exit, xpk_print
+
+
+def storage_create(args: Namespace) -> None:
+  k8s_api_client = setup_k8s_env(args)
+
+  install_storage_crd(k8s_api_client)
```

Comment:
I wonder if it wouldn't be better to have this only on cluster create instead of running it with each storage subcommand.

Reply:
I agree, done